### DISCUSSION TITLE
While importing meshes it gets messy

### DISCUSSION THREAD

**Discussion by Mehshida** [ComputationalBiomechanicsLab/opensim-creator#921] [Q&A] [answered]
When i try to import meshes in the opensimcreator to creat bodeies of rajagoapal, it results in this 
![Screenshot 2024-09-27 173201](https://github.com/user-attachments/assets/419263ed-3f21-48e5-bfda-c7bd437d5874)
how do i deal with this

**Comment by adamkewley** [comment, marked as answer]
The mesh importer imports meshes, with no further context about coordinate systems, bodies, etc. What you are visualizing is the meshes imported at `(0,0,0)` in the wider scene. It's up to you to place the bodies, joints, etc. in either the mesh importer or the model editor.

If you want to edit something in Rajagopal's coordinate system then you can either load the `.osim` in the model editor and edit the original model, or import it into the mesh importer via the `import` option, which will delete almost all information *apart* from the kinematic chain, body locations, etc.
